typespec/Applications.Core/examples/2023-10-01-preview/Gateways_Update.json

```diff
@@ -11,7 +11,7 @@
         "application": "/planes/radius/local/resourceGroups/testGroup/providers/Applications.Core/applications/app0",
         "routes":[
           {
-            "destination": "/planes/radius/local/resourcegroups/myawsenv/providers/applications.core/httproutes/gtwy-front-rte"
+            "destination": "http://frontend"
```

Comment:
Is this the address of the frontend container?

Reply:
Yes. This is an example btw.